python/fledge/services/core/api/control_service/script_management.py

```diff
@@ -143,6 +141,7 @@ async def add_schedule_and_configuration(request: web.Request) -> web.Response:
         raise web.HTTPBadRequest(reason=msg, body=json.dumps({"message": msg}))
     except Exception as ex:
         msg = str(ex)
+        _logger.error("Automation script schedule task failed. Found error: {}".format(msg))
```

Comment:
Task did not fail. 

Failed to add scheduled task for Automation script `<name>`

Reply:
Control or automation? Consistently use the same wording. 

May be Control automation